demo/universal/native/lib/Actions.re

```diff
@@ -0,0 +1,296 @@
+/*
+   Due to the issues that you can see at create (line 71), edit (line 136) and delete (line 192) functions we will have the args always as a list of a single list of args.
+   To parse it we need to get this first item that will indeed be the args and pass it further
+ */
+[@platform native]
+let getArgs = body => {
+  Yojson.Basic.from_string(body)
+  |> Yojson.Basic.Util.to_list
+  |> List.hd
+  |> Yojson.Basic.Util.to_list;
+};
+
+module Notes = {
+  [@platform native]
+  let createNoteResponse = note => {
+    React.Json(
```

Comment:
I think React.Json shouldn't be used and should be wrapped by ppx, do you agree?